lib/node_modules/@stdlib/blas/base/gscal/docs/types/index.d.ts

```diff
@@ -32,12 +32,12 @@ type InputArray = NumericArray | Collection<number> | AccessorArrayLike<number>;
 */
 interface Routine {
 	/**
-	* Multiplies a vector `x` by a constant `alpha`.
+	* Multiplies a vector `x` by a scalar constant `alpha`.
```

Comment:
```suggestion
	* Multiplies a vector by a scalar constant.
```
Same suggestion throughout this file.